api/pkg/filters/signature/main.go

```diff
@@ -0,0 +1,334 @@
+package main
```

Comment:
Will there be any sync required here after auth v4 changes https://github.com/sodafoundation/multi-cloud/pull/903/files#diff-dca9837476819ec12ee03315c11aca98 either here or in the other code?

Reply:
removed this file